crates/exec-wasmtime/src/lib.rs

```diff
@@ -148,11 +104,24 @@ mod test {
       (data (i32.const 0) "Hello, world!\0a")
     )"#;
 
+    pub fn run(wasm: &[u8]) -> anyhow::Result<Vec<Val>> {
+        let mut file = NamedTempFile::new().context("failed to create module file")?;
+        file.write(wasm).context("failed to write module to file")?;
+        file.rewind().context("failed to rewind file")?;
+        Runtime::execute(Package::Local {
+            #[cfg(unix)]
+            wasm: (&file).as_raw_fd(),
+            #[cfg(windows)]
+            wasm: file.into_file(),
```

Comment:
For future reference: this may create a race condition on Windows. If it does, we'll have to make the file persistent.
The docs are very unclear

Reply:
I think you must resolve your conversation with yourself.